### ISSUE TITLE
Accessibility issues reported by Firefox

### ISSUE THREAD

**Issue by albig** [verdigado/sunflower#853]
## Describe the bug

The Firefox Accessibility tool reports some minor issues:

1. [Form elements should have a visible text label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Text_labels_and_names#form_elements_should_have_a_visible_text_label) in the search box on top, right of Sunflower standard header.
2. Contrast error on text on cover image.

**Comment by albig** [comment]
closed with #854
